docker/scout/scripts/main.py

```diff
@@ -398,6 +452,21 @@ def main():
         documentation="Badgertree reward timestamp",
         labelnames=["lastCycleUnixtime"],
     )
+    #  --- ibBTC related ---
+    peak_gauge = Gauge(
+        name="peakcontracts",
+        documentation="Peak portfolio value",
+        labelnames=["peakContract", "tokenAddress", "param"],
+    )
+    ibBTC_contract_gauge = Gauge(
+        name="ibBTC", documentation="Interest bearing BTC token", labelnames=["param"]
+    )
+    ibBTC_composition_gauge = Gauge(
+        name="ibBTC_composition",
+        documentation="ibBTC composition",
+        labelnames=["peakContract", "peakName", "tokenName", "tokenAddress", "param"],
```

Comment:
Which token are you talking about?  Also we've used token everywhere else where there is a token name needed and grafana allows matching on names so use that standard.  If the token here is the underlying lp, maybe consider changing it to lptoken.  If it represents the underlying token (which is tricky in the case of crv) then it would probably make sense to add lptoken as well.  

again, things that have matching data should generally have matching var names across the entire promethues datasoruce (everything in all hte scripts)

Reply:
Changed `tokenName` to `token` to match the other gauges. Here, it's referring to the underlying Sett token.